src/main/java/com/alkemy/ong/controller/CommentController.java

```diff
@@ -21,7 +21,11 @@ public class CommentController {
 
     @GetMapping
     public ResponseEntity<?> listComments(){
-        return new ResponseEntity<>(commentsService.findAll(),HttpStatus.OK);
+        if(!commentsService.findAll().isEmpty()){
+             return new ResponseEntity<>(commentsService.findAll(),HttpStatus.OK);
+        }else{
+            return new ResponseEntity<>(messageHandler.listCommentsEmpty, HttpStatus.NOT_FOUND);
```

Comment:
banco la idea, pero el codigo no puede ser un 404 porque ese se arroja cuando buscas algo y no encontras, por lo general algo especifico, determinada ruta, determinada entidad. Aca la ejecucion es exitosa pero tenes una lista vacia, por lo que seria un json vacio para nosotros. Aca el codigo podria ser un 204 que significa NO CONTENT
https://apihandyman.io/empty-lists-http-status-code-200-vs-204-vs-404/#:~:text=The%20204%20(No%20Content)%20status,in%20the%20response%20payload%20body.&text=While%20200%20OK%20being%20a,is%20absolutely%20nothing%20to%20return.